### PR TITLE
Remove unrelevant information

### PR DESCRIPTION
**Pull request by coulof**
# Description
In an effort to simplify our doc I removed some of the verbiage

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by sharmilarama** [comment]
Do we want to completely remove all the pages for partners or can we consolidate that into a simple list?

**Review by mareksuski-dell — DISMISSED** [review]
LGTM but due to lack of long experiance with the CSM it is hard for me to validate if the "unrelevant information" was removed

File: content/docs/authorization/_index.md [hunk 1]
```diff
@@ -59,21 +59,8 @@ CSM for Authorization supports the following CSI drivers and versions.
 To resolve this, please refer to our [troubleshooting guide](./troubleshooting) on the topic.
 
 ## Authorization Components Support Matrix
-CSM for Authorization consists of 2 components - the Authorization sidecar and the Authorization proxy server.  It is important that the version of the Authorization sidecar image maps to a supported version of the Authorization proxy server.
+CSM for Authorization consists of 2 components - the Authorization sidecar bundled with the driver and the Authorization proxy server to validate the access. It is important to note that the sidecar is backward compatible with older proxy versions. Still, keeping the sidecar and proxy server with similar versions is advised.
```
[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/authorization/_index.md]: Addressed

**Review by shanmydell — DISMISSED** [review]

**Comment by coulof** [comment]
> LGTM but due to lack of long experiance with the CSM it is hard for me to validate if the "unrelevant information" was removed

The information was rather generic or redundant with what we have in the installation pages.

**Review by shanmydell — APPROVED** [review]

**Pull request merged**